scripts/run_e2e_tests.py

```diff
@@ -257,17 +262,22 @@ def setup_and_install_dependencies(skip_install):
         install_chrome_on_travis.main(args=[])
 
 
-def build_js_files(dev_mode_setting):
+def build_js_files(dev_mode_setting, deparallelize_terser=False):
     """Build the javascript files.
 
     Args:
         dev_mode_setting: bool. Represents whether to run the related commands
         in dev mode.
+        deparallelize_terser: bool. Represents whether to use webpack
+        compilation config that disables parallelism on terser plugin.
     """
     update_dev_mode_in_constants_js(CONSTANT_FILE_PATH, dev_mode_setting)
     if not dev_mode_setting:
         python_utils.PRINT('  Generating files for production mode...')
-        build.main(args=['--prod_env'])
+        if not deparallelize_terser:
+            build.main(args=['--prod_env'])
+        else:
+            build.main(args=['--prod_env', '--deparallelize_terser'])
```

Comment:
There is no need to negate the logic, just use:

```suggestion
        if deparallelize_terser:
            build.main(args=['--prod_env', '--deparallelize_terser'])
        else:
            build.main(args=['--prod_env'])
```